README.md

```diff
@@ -1 +1,23 @@
 # ext-userimport
+
+Make sure to upload your files containing user data to a secure folder!
+
+### Setup a protected storage folder
+
+1. Create a file protected "File Storage" record
+2. Create a folder /protected/userimport in your webroot
```

Comment:
> Create a folder /protected/userimport or similar in your webroot

I would switch item 1 and 2 because you want to create the file storage after the folder has been created.